### PR TITLE
Actually fix reordering and add new null tag type

### PR DESCRIPTION
**Pull request by CalebBredekamp**

### CODE REVIEW

**Comment by CalebBredekamp** [comment]
I didn't include the change to dtpw_export_view as I thought you probably have that edit on your side but let me know if you want me to add it.

File: Service/src/main/resources/dtpw/dtpw_views.sql [hunk 1]
```diff
@@ -320,27 +321,28 @@ SELECT
...
+    array(
+            SELECT unnest(tags) AS vals
+            FROM asset_tags WHERE asset_tags.asset_id = core.asset_id
+            ORDER BY core.asset_id, vals
+        ) AS tags,
 
     quantity.num_units AS extent,
     quantity.unit_code AS extent_unit,
-
+    core.last_changed,
```
Comment (frankvanrietimqs) [inline Service/src/main/resources/dtpw/dtpw_views.sql:339]: Remove this line, the field is not in master yet

**Review by frankvanrietimqs — COMMENTED** [review]
Nice work

**Comment by CalebBredekamp** [comment]
These changes are in the timestamp feature branch (Just in case I forget)

**Pull request closed without merging**